### PR TITLE
feat(trading,governance): improve flow of risk disclaimer presentation

### LINKED ISSUE #3737: Console in context messaging
Details in Miro

### PR DESCRIPTION
**Pull request by macqbat**
# Related issues 🔗

Closes #3737 

# Description ℹ️

Improve risk disclaimer experience

# Demo 📺

image
image
image
image

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: apps/trading/components/welcome-dialog/disclaimer.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { t } from '@vegaprotocol/i18n';
+
+export const Disclaimer = () => {
+  return (
+    <>
+      <p className="mb-6 mt-4">
+        {t(
+          'Vega is a decentralised peer-to-peer protocol that can be used to create liquidity and trade derivatives of cryptocurrencies. The Vega Protocol is an implementation layer (layer one) protocol made of free, public, open-source or source-available software. Use of the Vega Protocol involves various risks, including but not limited to, losses while digital assets are being supplied to the Vega Protocol and losses due to the fluctuation of prices of assets. Before using the Vega Protocol, you should review the relevant documentation to make sure that you understand how it works. dditionally, just as you can access email protocols such as SMTP through multiple email clients, ou can potentially access the Vega Protocol through dozens of web or mobile interfaces. You are responsible for doing your own diligence on those interfaces to understand the associated risks and any fees.'
```
Comment (candida-d) [inline apps/trading/components/welcome-dialog/disclaimer.tsx:8]:
```suggestion
          'Vega is a decentralised peer-to-peer protocol that can be used to create liquidity and trade derivatives of cryptocurrencies. The Vega Protocol is an implementation layer (layer one) protocol made of free, public, open-source or source-available software. Use of the Vega Protocol involves various risks, including but not limited to, losses while digital assets are being supplied to the Vega Protocol and losses due to the fluctuation of prices of assets. Before using the Vega Protocol, you should review the relevant documentation to make sure that you understand how it works. Additionally, just as you can access email protocols such as SMTP through multiple email clients, you can potentially access the Vega Protocol through dozens of web or mobile interfaces. You are responsible for doing your own diligence on those interfaces to understand the associated risks and any fees.'
```

File: apps/trading/components/welcome-dialog/disclaimer.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        )}
+      </p>
+      <p className="mb-8">
+        {t(
+          'This website is not located on the world wide web and its data is not stored on servers - physical or virtual. It is located on a decentralised network, the Interplanetary File System ("IPFS"). The IPFS decentralised web is made up of all the computers (nodes) connected to it. Data is therefore stored on many different computers. No party is the operator or hoster of this website. The information provided on this website does not constitute investment advice, financial advice, trading advice, or any other sort of advice and you should not treat any of the website\'s content as such. No party recommends that any cryptocurrency asset should be bought, sold, or held by you via this website. Do conduct your own due diligence and consult our financial advisor before making any investment decisions.'
```
Comment (candida-d) [inline apps/trading/components/welcome-dialog/disclaimer.tsx:23]:
```suggestion
          'This website is not located on the world wide web and its data is not stored on servers - physical or virtual. It is located on a decentralised network, the Interplanetary File System ("IPFS"). The IPFS decentralised web is made up of all the computers (nodes) connected to it. Data is therefore stored on many different computers. No party is the operator or host of this website. The information provided on this website does not constitute investment advice, financial advice, trading advice, or any other sort of advice and you should not treat any of the website\'s content as such. No party recommends that any cryptocurrency asset should be bought, sold, or held by you via this website. Do conduct your own due diligence and consult our financial advisor before making any investment decisions.'
```

Comment (candida-d) [inline apps/trading/components/welcome-dialog/disclaimer.tsx:23]:
```suggestion
          'This website is not located on the world wide web and its data is not stored on servers - physical or virtual. It is located on a decentralised network, the Interplanetary File System ("IPFS"). The IPFS decentralised web is made up of all the computers (nodes) connected to it. Data is therefore stored on many different computers. 
          
          No party is the operator or hoster of this website. The information provided on this website does not constitute investment advice, financial advice, trading advice, or any other sort of advice and you should not treat any of the website\'s content as such. No party recommends that any cryptoasset should be bought, sold, or held by you via this website. No party ensures the accuracy of information listed on this website or holds any responsibility for any missing or wrong information. You understand that you are using any and all information available here at your own risk.'
```

File: apps/trading-e2e/src/integration/navbar.cy.ts [hunk 3]
```diff
@@ -47,6 +47,26 @@ describe('Navbar', { tags: '@smoke' }, () => {
...
+      cy.getByTestId('Disclaimer').eq(0).click();
+      cy.getByTestId('dialog-content').should('be.visible');
+      cy.getByTestId('dialog-content')
+        .eq(0)
+        .within(() => {
+          cy.getByTestId('dialog-title').contains('Disclaimer');
+          cy.get('p')
+            .eq(0)
+            .contains(
+              'Vega is a decentralised peer-to-peer protocol that can be used to create liquidity and trade derivatives of cryptocurrencies.'
```
Comment (candida-d) [inline apps/trading-e2e/src/integration/navbar.cy.ts:65]:
```suggestion
              'Vega is a decentralised peer-to-peer protocol that can be used to trade derivatives with cryptoassets.'
```

File: apps/trading/components/welcome-dialog/risk-notice-dialog.tsx [hunk 4]
```diff
@@ -88,29 +86,75 @@ const TestnetContent = ({ network }: { network: Networks }) => {
...
-      </p>
-      <h4 className="text-xl mb-2">
-        {t('Technical and financial risk of loss')}
-      </h4>
+    <div className="mt-6">
+      <div className="bg-neutral-100 dark:bg-neutral-800 dark:text-neutral-200 p-6 mb-6">
+        <ul className="list-disc ml-6 text-lg">
+          <li>
+            {t(
+              'No party hosts or operates this IFPS website or offers any financial advice.'
```
Comment (candida-d) [inline apps/trading/components/welcome-dialog/risk-notice-dialog.tsx:121]: This sentence has been cut so it would be worth reconsidering what goes in here

**Review by candida-d — COMMENTED** [review]
I made some suggestions that are based on changes to the source document.

File: apps/trading/components/welcome-dialog/disclaimer.tsx (hunk 1 above)
Comment (macqbat) [reply apps/trading/components/welcome-dialog/disclaimer.tsx]: text has been changed according given pattern

File: apps/trading-e2e/src/integration/navbar.cy.ts (hunk 3 above)
Comment (macqbat) [reply apps/trading-e2e/src/integration/navbar.cy.ts:65]: text has been changed according given pattern

File: apps/trading/components/welcome-dialog/risk-notice-dialog.tsx (hunk 4 above)
Comment (macqbat) [reply apps/trading/components/welcome-dialog/risk-notice-dialog.tsx]: text has been changed according given pattern

File: apps/trading/components/welcome-dialog/disclaimer.tsx (hunk 2 above)
Comment (macqbat) [reply apps/trading/components/welcome-dialog/disclaimer.tsx]: text has been changed according given pattern

**Review by bglownia — APPROVED** [review]

File: apps/trading/components/welcome-dialog/risk-message.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+import { Links, Routes } from '../../pages/client-router';
+
+export const RiskMessage = () => {
+  return (
+    <>
+      <div className="bg-neutral-100 dark:bg-neutral-800 dark:text-neutral-200 p-6 mb-6">
+        <ul className="list-disc ml-6 text-lg">
+          <li>
+            {t(
+              'No party hosts or operates this IFPS website or offers any financial advice.'
```
Comment (candida-d) [inline apps/trading/components/welcome-dialog/risk-message.tsx:12]: This has been cut from the text. Perhaps you can replace it with the following, depending on where it turns up/what it's for: 
 "This website is hosted on a decentralised network, the Interplanetary File System (“IPFS”).  The IPFS decentralised web is made up of all the computers (nodes) connected to it.  Data is therefore stored on many different computers."

Comment (MadalinaRaicu) [inline apps/trading/components/welcome-dialog/risk-message.tsx:12]:
```suggestion
              'This website is hosted on a decentralised network, the Interplanetary File System (“IPFS”). The IPFS decentralised web is made up of all the computers (nodes) connected to it. Data is therefore stored on many different computers.'
```

**Review by sam-keen — APPROVED** [review]

File: apps/trading/components/welcome-dialog/risk-message.tsx (hunk 5 above)
Comment (mattrussell36) [reply apps/trading/components/welcome-dialog/risk-message.tsx]: Removing this line as discussed

Comment (mattrussell36) [reply apps/trading/components/welcome-dialog/risk-message.tsx]: Removing this line as discussed

**Pull request merged**